test/fixtures/kitchen/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./tsconfig-google.json",
+  "extends": "./node_modules/google-ts-style/tsconfig-google.json",
```

Comment:
This is a system test. It installs ts-style locally. For this system test, this is the correct system path. Did you have a more specific question?

Note that this system test needs to be expanded a lot more. It should go through the exercise of creating a temp directory out-of-tree (so that we don't inherit any node_modules from the parent folder), running `google-ts-style init`, dropping in some TS code and then making sure everything is setup correctly for the project.